src/app.scss

```
@@ -0,0 +1,14 @@
+:root {
+  --container-gap: 5rem;
+}
+
+.app_container {
+  width: 80%;
+  margin: 0 auto;
+  display: flex;
+  flex-direction: column;
+  gap: var(--container-gap);
+}
```

Can we sort the attributes alphabetically? It would be great if you can do this for every block throughout the App.
```suggestion
.app_container {
  display: flex;
  flex-direction: column;
  gap: var(--container-gap);
  margin: 0 auto;
  width: 80%;
}
```